### DISCUSSION TITLE
Changing workspace names

### DISCUSSION THREAD

**Discussion by win8linux** [Floorp-Projects/Floorp#764] [Feature Request]
The new workspace implementation does not have a way to change names, not in the settings nor even while creating a new workspace. If there is one, I'd like to know where and maybe turn this into a UX issue afterwards.

**Comment by win8linux** [comment]
It turns out that there is a way to rename workspaces, however you need to turn on `Manage Workspace on Browser Manager Sidebar`, then right-click on a workspace. There should be an alternate way to change workspace names for people who do not manage workspaces via the sidebar, which at the moment is the default when enabling workspaces. Additionally, upon creating workspaces there should be a prompt to enter a name first.

**Reply by CutterKnife** [reply]
Right-click on the workspace and select "Rename Workspace". This is also the same in the toolbar popup.
![スクリーンショット 2024-01-23 15 57 04](https://github.com/Floorp-Projects/Floorp/assets/110194318/1f1759bf-f638-4938-973c-072f2edb8037)
